Makefile

```diff
@@ -46,3 +46,13 @@ yq:
 pint_lint:
 	echo "Linting Prometheus rules..."
 	./pint --no-color lint ${EXTRACTED_RULE_FILE}
+
+yamllint:
+	echo "Installing yamllint..."
+	pip install pipenv
+	pipenv sync --dev --skip-lock
```

Comment:
We should just sync with an existing lock.
In any case, I don't think it will help us to have it run here. We cannot run it in advance.
The CI is running each of the steps of the pull-request.yaml on a different image. They share the working directory but nothing else. It means, that we can fetch files on one step and use it on the other, but we cannot load a python virtual env to memory on one container and use it on another.

All in all, at least in CI, we must load the virtual env and use it on the same step. It can be on different makefile targets if it will help us sharing steps as much as possible between dev and ci, but it cannot happen on different steps.